src/characters/cbd-recipient.ts

```diff
@@ -90,21 +82,25 @@ export class CbdTDecDecrypter {
       provider,
       this.ritualId
     );
+    // We only need the `threshold` participants
+    const sufficientDkgParticipants = dkgParticipants.slice(0, this.threshold);
     const contextStr = await conditionExpr.buildContext(provider).toJson();
     const { sharedSecrets, encryptedRequests } = this.makeDecryptionRequests(
       this.ritualId,
       variant,
       ciphertext,
       conditionExpr,
       contextStr,
-      dkgParticipants
+      sufficientDkgParticipants
```

Comment:
Is this limiting the number of Ursulas that we send the decryption request to, to be only a threshold number?If so, we need a `threshold` to *respond*, but we should still send requests to more than the threshold, or else we are expecting no non-responsive Ursulas. Porter takes care of ensuring a threshold number of responses.

Reply:
Agreed - let's call that a _robustness enhancement_ and track it in an issue.